<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>767</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>235</green>
        <blue>216</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>235</green>
        <blue>216</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>235</green>
        <blue>216</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>235</green>
        <blue>216</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>SmartCrawling</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="category_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sort_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white</string>
    </property>
    <item>
     <property name="text">
      <string>인기도순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>누적판매순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>낮은가격순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>최신등록순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>리뷰많은순</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>평점높은순</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="link_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>https://사이트주소</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="as_num_edit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="as_info_edit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="price_edit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QTableView" name="dataframe_table">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>951</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(202, 196, 172);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
   </widget>
   <widget class="QComboBox" name="method_edit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <item>
     <property name="text">
      <string>원</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="ban_edit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>크롤링 시작하기</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="remove_edit">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>320</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>610</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>EXCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>홈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>HTML</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>스마트스토어 상품 불러오기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>값 설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>금지어 설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>A/S 전화번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>A/S 안내내용</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>판매가 추가금액</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>260</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>삭제 행 선택</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="storage_edit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>650</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;폴더경로\파일이름.xlsx&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_edit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 117, 117);
color:white;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>html_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>769</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>crawling_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>step0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>781</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>replace_as()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>828</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>replace_ban()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1124</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>953</x>
     <y>770</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1074</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>1015</x>
     <y>780</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>final()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1148</x>
     <y>677</y>
    </hint>
    <hint type="destinationlabel">
     <x>1172</x>
     <y>783</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>file_find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>788</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>html_settings()</slot>
  <slot>crawling_start()</slot>
  <slot>step0()</slot>
  <slot>replace_ban()</slot>
  <slot>replace_as()</slot>
  <slot>remove_row()</slot>
  <slot>final()</slot>
  <slot>file_find()</slot>
 </slots>
</ui>
